src/rgw/rgw_process.cc

```diff
@@ -320,6 +320,10 @@ int process_request(rgw::sal::RGWRadosStore* const store,
 	  << " op status=" << op_ret
 	  << " http_status=" << s->err.http_ret
 	  << " latency=" << s->time_elapsed()
+	  << " op_method=" << s->info.method
+	  << " op_target=" << s->info.request_uri
+	  << " op_content_length=" << s->content_length
+	  << " op_user=" << s->user
```

Comment:
where does the `op_` part come from?

what does content length mean here? is it the length of the request or the response?

Reply:
The `op_` part comes from the `op status` in this log line. It could be removed. 

As for the content length here is referring to the length of the request.

Reply:
I'm not sure the request length is that interesting, but also intuitively I would have thought this would be the response length, so perhaps worth changing the name a bit. (op_request_content_length?)